Add revamped documentation to iso8583 and smpp connectors (#1607)
wso2/docs-mi: en/docs/reference/connectors/smpp-connector/smpp-connector-example.md

### Configure the API

> 1. First, we will create the `/send` resource to send an SMS to the Short Message Service Center. Refer the [Adding new API resources]({{base_path}}/develop/creating-artifacts/creating-an-api/#adding-new-api-resources) guide to create a new resource. Provide the resource details as below.

Author: Fixed

    - **URI Template**: `/send`
    - **HTTP Method**: `POST`

2. Set up the **sendSMS** operation.

>     1. Select the **send** resource. Click on the **+** mark indicated below to go to the **SMPP** connector and select **sendSMS** operation. 

Reviewer: ```suggestion
    1. Select the **send** resource. Click on the **+** icon on the canvas to open the **Mediator Palette**, search for the **SMPP** connector, and select the **sendSMS** operation.
```

        
        While invoking the API, the values of the above three parameters come from user input.
    
    4. Add the `sendSMS` operation in the `SMPP` connector. Select the previously created connection. Provide the following expressions for source address, destination address, and message as below. 
        - **Source Address** - expression `payload.sourceAddress`
        - **Destination Address** - expression `payload.destinationAddress` 
        - **Message** - expression `payload.message`

>         To store the response of the operation in the message context, select "Overwrite Message Body" in the Output Section. This will allow you to send the response back to the user as a response of the API invocation.

Reviewer: ```suggestion
        To store the response of the operation in the message body, select **Overwrite Message Body** in the Output Section. This will allow you to send the response back to the user as a response of the API invocation.
```

Reviewer: In the sample XML config this is set to false. https://github.com/wso2/docs-mi/pull/1607/files#diff-62d2e42e83e128a0f4c5a805d158fd9472ea69a8a75f2f070ed3f9b02d3b690bR90. Please check and update.

> 3. Get a response from the user.          

Reviewer: ```suggestion
3. Send a response to the user.          
```
